src/components/cylc/tree/deltas.js

```diff
@@ -213,11 +214,13 @@ function handleDeltas (deltas, workflow, lookup, options) {
  */
 export default function (data, workflow, lookup, options) {
   const deltas = data.deltas
-  // first we check whether it is a shutdown response
-  if (deltas.shutdown) {
-    CylcTree.clear(workflow)
-    return {
-      errors: []
+  // first we check whether it is a new start
+  if (deltas && deltas.added && deltas.added.workflow) {
+    if (deltas.added.workflow.status === WorkflowState.RUNNING.name) {
+      // The workflow could be stopped. In this case when restarted (cold or hot)
+      // it would be hard to apply the new deltas keeping the UI state valid. So
+      // for now we clear the tree, and start from scratch rebuilding it.
+      CylcTree.clear(workflow)
```

Comment:
Note too that at the moment the tree view has no reaction when the workflow status change. We could gray-out the whole area, disable the tree, set an icon somewhere, etc. But I think that will be done in a follow-up, or maybe part of #332

Reply:
I think the background colour could change if the workflow is stopped just to make it clear that the data is offline. But yes, work for later.

(experience with Cylc Review has show us that people look past the workflow status in the top left and often interpret a running task to mean a running workflow)